auction-server/src/state.rs

```diff
@@ -845,16 +816,22 @@ pub struct ExpressRelaySvm {
 }
 
 pub struct Store {
-    pub chains:            HashMap<ChainId, ChainStoreEvm>,
-    pub chains_svm:        HashMap<ChainId, ChainStoreSvm>,
-    pub event_sender:      broadcast::Sender<UpdateEvent>,
-    pub opportunity_store: OpportunityStore,
-    pub ws:                WsState,
-    pub db:                sqlx::PgPool,
-    pub task_tracker:      TaskTracker,
-    pub secret_key:        String,
-    pub access_tokens:     RwLock<HashMap<models::AccessTokenToken, models::Profile>>,
-    pub metrics_recorder:  PrometheusHandle,
+    pub chains:           HashMap<ChainId, ChainStoreEvm>,
+    pub chains_svm:       HashMap<ChainId, ChainStoreSvm>,
+    pub event_sender:     broadcast::Sender<UpdateEvent>,
+    pub ws:               WsState,
+    pub db:               sqlx::PgPool,
+    pub task_tracker:     TaskTracker,
+    pub secret_key:       String,
+    pub access_tokens:    RwLock<HashMap<models::AccessTokenToken, models::Profile>>,
+    pub metrics_recorder: PrometheusHandle,
+}
+
+#[derive(Clone)]
+pub struct StoreNew {
+    pub opportunity_service_evm:
+        Arc<opportunity_service::Service<opportunity_service::ChainTypeEvm>>,
```

Comment:
We need to call sth in the auction module, so we need to access the store itself within the service.
When we refactor other parts, this will be fixed and we'll have only a single store with services in it.
So you are right, this isn't sth that I like to have in the final design, however we need it for now to take care of this call